platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/datasource/VirtualDataSourceBuilder.java

```diff
@@ -40,10 +41,11 @@ public interface VirtualDataSourceBuilder extends SelfSerializable {
      * @param withDbCompactionEnabled
      * 		If true then the new database will have background compaction enabled, false and the
      * 		new database will not have background compaction enabled
+     * @param configuration platform configuration
      * @return
      * 		An opened {@link VirtualDataSource}.
      */
-    VirtualDataSource build(String label, final boolean withDbCompactionEnabled);
+    VirtualDataSource build(String label, final boolean withDbCompactionEnabled, Configuration configuration);
```

Comment:
Should it be a part of `VirtualDataSourceBuilder` interface or rather implementation details? I mean, when an instance of a builder like `MerkleDbDataSourceBuilder` is created, it gets a reference to `Configuration`, which is then used in all interface methods

Reply:
Addressed, see [this commit](https://github.com/hashgraph/hedera-services/pull/15562/commits/8c39dd8d07d57ba39218920cf5dd158000d4ab7e)